cmd/restic/cmd_stats.go

```diff
@@ -48,8 +55,8 @@ Refer to the online manual for more details about each mode.
 func init() {
 	cmdRoot.AddCommand(cmdStats)
 	f := cmdStats.Flags()
-	f.StringVar(&countMode, "mode", countModeRestoreSize, "counting mode: restore-size (default), files-by-contents, blobs-per-file, or raw-data")
-	f.StringVarP(&snapshotByHost, "host", "H", "", "filter latest snapshot by this hostname")
+	f.StringVar(&countMode, "mode", countModeRestoreSize, "counting mode: restore-size (default), files-by-contents, blobs-per-file, raw-data or repository")
+	f.StringVarP(&snapshotByHost, "host", "H", "", "filter latest snapshot by this hostname (not used for mode repository)")
```

Comment:
Instead of ignoring the flag, wouldn't it be better to check it and bail if it's set?

Reply:
no longer relevant, as I put this into a new command `repoinfo`.